docs/developer-guide/ksqldb-reference/aggregate-functions.md

```diff
@@ -31,12 +31,14 @@ Stream, Table
 Return an array containing all the values of `col1` from each
 input row (for the specified grouping and time window, if any).
 Currently only works for simple types (not Map, Array, or Struct).
-This version limits the size of the result Array to a maximum of
-1000 entries and any values beyond this limit are silently ignored.
+
+The size of the result Array can be limited to a maximum of
+`ksql.functions.collect_list.limit` entries and any values beyond this 

```

Comment:
I think these params also need to be documented in https://docs.ksqldb.io/en/latest/reference/server-configuration/